tests/ldap_test.ldif

```diff
@@ -12,3 +12,37 @@ gidNumber: 1001
 userPassword: 123456
 mail: test@example.com
 gecos: test
+
```

Comment:
could you add some comments? Otherwise, it's hard for others to modify without background.

Reply:
Updated